docs/Getting_Started.md

```diff
@@ -0,0 +1,394 @@
+2. Adds a new game to the `activeGames` object using the interaction ID. The active game records the `userId` and `objectName`.
+3. Sends a message back to the channel with a button with a `custom_id` of `accept_button_<SOME_ID>`.
+
+> warn
+> The sample code uses an object as in-memory storage, but for production apps it's recommended to use a database.
+
+When sending a message with [message components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/what-is-a-component), the components' payloads are added to a `components` array. Actionable components (like buttons) need to be inside of an [action row](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/action-rows), which you can see in the code sample.
+
+Note the unique `custom_id` sent with message components, in this case `accept_button_` with the active game's ID appended to it. A `custom_id` can be used to handle requests that Discord sends you when someone interacts with the component, which you'll see in a moment.
+
+Now when you run the `/challenge` command and pick an option, your app will send a message with an **Accept** button, though we still need to add the code to handle it.
+
+### Handling buttons
+
+When users interact with one of your app's message components, Discord will send a request with an [interaction type](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-type) of `3` (or the  `MESSAGE_COMPONENT` value when using `discord-interactions`).
+
+To set up a handler for the button, we’ll check the `type` of interaction, followed by matching the `custom_id`. Paste the following code under the type handler for `APPLICATION_COMMAND`s:
+
+```javascript
+if (type === InteractionType.MESSAGE_COMPONENT){
+    // custom_id set in payload when sending message component
+    let componentId = data.custom_id;
+
+    if (componentId.startsWith('accept_button_')) {
+        // get the associated game ID
+        let gameId = componentId.replace('accept_button_', '');
+        // Delete message with token in request body
+        let url = DiscordAPI(`/webhooks/${process.env.APP_ID}/${req.body.token}/messages/${req.body.message.id}`);
+        client({ url, method: 'delete' }).catch(e => console.error(`Error deleting message: ${e}`));
+
+        return res.send({
+            "type": InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
+            "data": {
+                // Fetches a random emoji to send from a helper function
+                "content": "What's your object of choice?",
+                // Indicates it'll be an ephemeral message
+                "flags": 64,
+                "components": [{
+                    "type": ComponentType.ACTION,
+                    "components": [{
+                        "type": ComponentType.SELECT,
+                        // Append game ID
+                        "custom_id": `select_choice_${gameId}`,
+                        "options": getShuffledOptions()
```

Comment:
```suggestion
            type: InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
            data: {
                // Fetches a random emoji to send from a helper function
                content: "What's your object of choice?",
                // Indicates it'll be an ephemeral message
                flags: 64,
                components: [{
                    type: ComponentType.ACTION,
                    components: [{
                        type: ComponentType.SELECT,
                        // Append game ID
                        custom_id: `select_choice_${gameId}`,
                        options: getShuffledOptions()
```

Reply:
These are intended to JSON payloads, which use double quotes around key and value